incubator/eventrouter/values.yaml

```diff
@@ -0,0 +1,8 @@
+# Default values for eventrouter.
+# This is a YAML-formatted file.
+# Declare name/value pairs to be passed into your templates.
+
+namespace: "kube-system"
```

Comment:
This isn't actually used, and Helm only allows you to specify the namespace using `helm install --namespace`, so I think this needs to be removed. If this needs to be installed into kube-system, you could add a note in the README or NOTES.txt